lib/liveValidation/liveValidatorLoader.ts

```diff
@@ -88,24 +87,16 @@ export class LiveValidatorLoader implements Loader<SwaggerSpec> {
     }
   );
 
-  public static create = getLoaderBuilder(
-    (opts: LiveValidatorLoaderOptions) => new LiveValidatorLoader(opts)
-  );
-  private constructor(private opts: LiveValidatorLoaderOptions) {
+  public constructor(
+    @inject(TYPES.opts) private opts: LiveValidatorLoaderOption,
+    private jsonLoader: JsonLoader,
+    private swaggerLoader: SwaggerLoader
+  ) {
     setDefaultOpts(opts, {
       transformToNewSchemaFormat: false,
-      loadSuppression: Object.keys(allErrorConstants),
     });
 
-    this.jsonLoader = JsonLoader.create(opts);
```

Comment:
This needs to be added.

Reply:
This one is converted to reflect injection so could be removed

Reply:
Just pass `schemaValidationOption` to `AjvSchemaValidator`